database/README.md

```diff
@@ -1,14 +1,414 @@
+
+ Column        | Type   | Nullable | Details
+---------------|--------|----------|--------
+ order\_uid    | bytea  | not null | order that got refunded
+ block\_number | bigint | not null | in which block the order got refunded
+ tx\_hash      | bytea  | not null | hash of the transaction that refunded the order
+
+Indexes:  
+- PRIMARY KEY: btree(`order_uid`)  
+
+### flyway\_schema\_history
+
+We use flyway to do migrations of our database schema. This table contains metadata for flyway to know which and when migrations have been applied. Since this table only contains data managed by flyway and we didn't encounter any need to take a closer look at it we'll just refer to the [flyway docs](https://flywaydb.org/documentation/).
+
+### interactions
+
+The settlement contract allows associating user provided interactions to be executed before and after an order. This table stores these interactions and associates them with the respective orders.
+
+ Column     | Type                   | Nullable | Details
+------------|------------------------|----------|--------
+ order\_uid | bytea                  | not null | order that this interaction belongs to
+ index      | integer                | not null | index indicating in which interactions should be executed in case the same order has multiple interactions (ascending order)
+ target     | bytea                  | not null | address of the smart contract this interaction should call
+ value      | numeric                | not null | amount of ETH this interaction should send to the smart contract
+ data       | bytea                  | not null | call data that contains the function selector and the bytes passed to it
+ execution  | [enum](#executiontime) | not null | in which phase the interaction should be executed
+
+Indexes:  
+- PRIMARY KEY: btree(`order_uid`)  
+
+
+### invalidations
+
+Stores data of [`OrderInvalidated`](https://github.com/cowprotocol/contracts/blob/main/src/contracts/GPv2Settlement.sol#L70-L71) events emited by [`invalidateOrder()`](https://github.com/cowprotocol/contracts/blob/main/src/contracts/GPv2Settlement.sol#L244-L255) of the settlement contract.
+
+ Column        | Type   | Nullable | Details
+---------------|--------|----------|--------
+ block\_number | bigint | not null | block in which the event was emitted
+ log\_index    | bigint | not null | index in which the log was emitted
+ order\_uid    | byteai | not null | order that got invalidated
+
+Indexes:  
+- PRIMARY KEY: btree(`block_number, log_index`)  
+- invalidations\_order\_uid: btree(`order_uid`, `block_number`, `log_index`)  
+
+### onchain\_order\_invalidations
+
+Stores data of [`OrderInvalidation`](https://github.com/cowprotocol/ethflowcontract/blob/main/src/interfaces/ICoWSwapOnchainOrders.sol#L46-L49) events emited by the `ICoWSwapOnchainOrders` interface.
```

Comment:
```suggestion
Stores data of [`OrderInvalidation`](https://github.com/cowprotocol/ethflowcontract/blob/main/src/interfaces/ICoWSwapOnchainOrders.sol#L46-L49) events emitted by the `ICoWSwapOnchainOrders` interface.
```
(A small but useful tip: if you're writing lots of text, install a spellchecker in your IDE. It's a lifesaver.)